src/LaravelMatomoTracker.php

```diff
@@ -138,7 +138,7 @@ public function setQueue(string $queueName)
      *
      * @return $this
      */
-    public function setCustomDimension(int $customDimensionId, string $value)
+    public function setCustomDimension($id, $value)
     {
         $this->setCustomTrackingParameter('dimension' . $customDimensionId, $value);
```

Comment:
The variable $customDimensionId needs to be renamed to $id as well.

Reply:
Just seen the comment, but you have fixed it anyway right? Thanks for merging!